### ISSUE TITLE
Evaluate grafana/loki as logging backend

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/gardener#918]
Currently cluster level logging and monitoring for seed clusters are handled by two different systems - the end user/operator uses Grafana to deal with metrics and Kibana to deal with logs.

### Motivation

- Metrics and logs are part of incident resolving and usually you need both of them. Now there is a context switching from one system to another one. loki promises to minimize this cost of context switching and to allow easy switch between metrics and logs for given search query.
- It is not easy to operate and scale Elasticsearch. It is resource intensive and requires not a few memory. loki promises to be lightweight (cost effective) and easy to operate.
- Grafana has native support for loki. Grafana also supports Elasticsearch datasource.

### Outcome

The idea of this issue is to evaluate loki as logging backend (more details about the requirements could be found in #3). As an outcome, implementation proposal (PR) or a list of limitations, issues of loki could be accepted.

---

- [Loki Design Doc](https://docs.google.com/document/d/11tjK_lvp1-SVsFZjgOTr1vV3-q6vBAsZYIQ5ZeYBkyM/view#heading=h.5xmknwtxwpne)

**Comment by vlerenc** [comment]
FYI: cc @dkistner @wyb1

**Comment by SayakMukhopadhyay** [comment]
Hi,
I noticed that #2175 was closed in favour of this and so I had a few questions regarding the roadmap.

1. Right now, the logging/monitoring stack consists of prometheus/grafana and fluentd/kibana. Is this going to replace the fluentd/kibana with loki/grafana?
2. Is OIDC as mentioned in #999 in the pipeline with this?
3. I was planning to try and contribute to #999 . Should I hold it off for now and wait until this closes?
